File: arrays/Python/Palindrome Number/Palindrome-Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        num = x
        if x < 0:
            return False

        reverseNum = 0
        while x > 0:
            remainder = x % 10
            reverseNum = reverseNum * 10 + remainder
            x //= 10

        return num == reverseNum

File: arrays/Palindrome-Number.cpp
class Solution {
    public boolean isPalindrome(int x) {
        int num=x;
        if(x<0)
            return false;
        
        int reversenum=0;
        while(x>0){
            int remainder = x%10;
            reversenum = reversenum * 10 + remainder;
            x/=10;
        }
        return num == reversenum;

        
    }
}
